web/src/lib/utils/helpers.ts

```diff
@@ -246,3 +246,32 @@ export const getConfigFieldByLegalStructure = (
 export const openInNewTab = (url: string): void => {
   window.open(url, "_blank", "noopener noreferrer");
 };
+
+export const removeMarkdownFormatting = (markdownText: string): string => {
+  return (
+    markdownText
+      // Remove headers
+      .replaceAll(/(^|\s)(#{1,6})\s/g, "$1")
+      // Remove images
+      .replaceAll(/!\[.*?]\(.*?\)/g, "")
+      // Remove links
+      .replaceAll(/\[.*?]\(.*?\)/g, "")
+      // Remove strong and em tags
+      .replaceAll(/(\*\*|__)(.*?)\1/g, "$2")
+      .replaceAll(/(\*|_)(.*?)\1/g, "$2")
+      // Remove inline code
+      .replaceAll(/`{1,2}[^`]*`{1,2}/g, "")
+      // Remove blockquotes
+      .replaceAll(/^\s*>\s?/gm, "")
+      // Remove unordered list bullets
+      .replaceAll(/^\s*[*+-]\s/gm, "")
+      // Remove ordered list numbers
+      .replaceAll(/^\s*\d+\.\s/gm, "")
+      // Remove horizontal rules
+      .replaceAll(/^-{3,}\s*$/gm, "")
+      // Remove extra lines
+      .replaceAll(/\n{2,}/g, " ")
+      // Remove all remaining line breaks
+      .replaceAll(/(\r\n|\n|\r)/gm, "")
+  );
+};
```

Comment:
Most of this is unused, but the idea is that it could be easily reference to comprehensively strip formatting in the future.